.github/workflows/tests.yaml

```diff
@@ -92,7 +92,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-10.15, windows-2019]
-        python-version: [3.8, 3.9]
+        python-version: [3.8, 3.9, 3.10]
```

Comment:
Needs to be "3.10" - as YAML converts it to a float otherwise.

Reply:
Thanks. I added it to craft-parts but I forgot to do the same thing here.